list.php

```diff
@@ -49,7 +55,14 @@ function getFullName() {
 </div>
 
 <!-- Domain List -->
+<?php if($list === "black") { ?>
+<h3>Exact blocking</h3>
+<?php } ?>
 <ul class="list-group" id="list"></ul>
+<?php if($list === "black") { ?>
+<h3>Wildcard blocking</h3>
+<ul class="list-group" id="list-wildcard"></ul>
+<?php } ?>
```

Comment:
The headings don't show up unless the page is reloaded. Maybe dynamically change it via JS.